gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v4_5_0/08_add_scoring_ruleset_table.yml

```diff
@@ -0,0 +1,26 @@
+databaseChangeLog:
+  - changeSet:
+      id: 4.5.0_add_scoring_rulesets
+      author: GraviteeSource Team
```

Comment:
### :warning: Aqua detected sensitive data in your code  
**Category:** Env Generic   
**Description:** Env Generic Credential   
**Severity:** HIGH   
**Match:**       author: *******************  
[This comment was created by Aqua Pipeline]